apps/browser/src/autofill/content/autofill.js

```diff
@@ -993,11 +993,18 @@
       function fillTheElement(el, op) {
           var shouldCheck;
           if (el && null !== op && void 0 !== op && !(el.disabled || el.a || el.readOnly)) {
-              const tabURLChanged = !fillScript.savedUrls?.some(url => url.startsWith(window.location.origin))
+              let tabURLChanged = !fillScript.savedUrls?.some(url => url.startsWith(window.location.origin));
+
+              // If the page origin doesn't match a savedURL, check the page location against equivalent domains
+              if (tabURLChanged) {
+                tabURLChanged = !fillScript.equivalentDomains.some(url => window.location.host.endsWith(url));
```

Comment:
```suggestion
                tabURLChanged = !fillScript.equivalentDomains?.some(url => window.location.host.endsWith(url));
```

Is there a possibility here that `equivalentDomains` is undefined?

Reply:
It shouldn't be - the reducer in `generateLoginFillScript` should ensure an empty array at minimum